### DISCUSSION TITLE
How to see exact bank balance?

### DISCUSSION THREAD

**Discussion by Fasidious** [endless-sky/endless-sky#9657] [Q&A] [answered]
Hello!

Is there a way to see your exact bank balance once you get into the millions of credits?

If not, could it be shown on the Bank screen?

**Comment by Koranir** [comment]
> Hello!
> 
> Is there a way to see your exact bank balance once you get into the millions of credits?
> 
> If not, could it be shown on the Bank screen?

You could always check your save file for the exact number :)

**Reply by Fasidious** [reply]
Sure, but it would be much better to be able to see it in-game.

**Comment by mOctave** [comment, marked as answer]
No, there is not a way to see your exact bank balance in game.

There are a few workarounds. For example, you could go and buy millions of units of ammo from an outfitter into storage at a known rate, then sell them back. A little math and an awful lot of effort, and that would give you your balance.

Alternatively, as Koranir said, you could just check your save. The credit balance appears fairly near the top, and isn't too hard to find.

As for showing it on the bank screen, I don't see why not. After all, if anyone got enough money to overflow that screen, they'd have other problems. I suggest you open an issue for it.

Overall, though, I don't think the exact bank balance is a big issue. You're unlikely to _need_ to know it, and ES is generally much more about the story than about making as much money as possible.

**Reply by Fasidious** [reply]
Thanks. 
Yeah, I don’t _need_ to know my exact balance, but the data control freak in me really _really_ wants to.
I think I will suggest exact bank balance in bank screen as a feature.
